tests/mocha/003-tasks.js

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright (c) 2014-2015 Digital Bazaar, Inc. All rights reserved.
+ */
+
+'use strict';
+
+var async = require('async');
+var bedrock = require('bedrock');
+var config = bedrock.config;
+var request = require('request');
+var jar = null;
```

Comment:
Looking at the rest of the file, it doesn't appear necessary to make this var file-scoped (keep it local to use).